app-frontend/src/app/pages/settings/profile/profile.html

```diff
@@ -12,14 +12,19 @@ <h1 class="h3 page-title">Profile</h1>
         </div>
         <div class="picture-description">
           <div class="panel panel-info">
-            <div class="panel-body">
-              To change your profile picture, upload a new one to your linked {{$ctrl.providerName}} account
+            <div class="panel-body" ng-if="!($ctrl.provider.provider === 'gravatar' || $ctrl.provider.provider === 'auth0')">
+              To change your profile picture, upload a new one to your linked
+              <a ng-href="{{$ctrl.provider.link}}" target="_blank">{{$ctrl.provider.name}}</a> account.
+            </div>
+            <div class="panel-body" ng-if="$ctrl.provider.provider === 'gravatar' || $ctrl.provider.provider === 'auth0'">
+              This profile picture is associated with your email in Gravatar. Please find out about how to change it
```

Comment:
I'd word it as `Your profile picture is managed on Gravatar. [Click Here for instructions on how to change it]` (whole sentence as link to make it easy to click)